scripts/apps/contacts/helpers.ts

```diff
@@ -85,19 +86,29 @@ function contactHasEmailAddress(contact: IContact): boolean {
 
 export function validateMinRequiredField(contact: IContact): boolean {
     return _.some(LOOKUP_FIELDS, (field) => {
-        switch (field) {
+        switch (field.name) {
         case 'contact_email':
             return contactHasEmailAddress(contact);
         case 'contact_phone':
         case 'mobile':
-            return _.get(contact, field, []).length > 0 &&
-                !_.some(_.map(contact[field], 'number'), (v) => _.isEmpty(v));
+            return _.get(contact, field.name, []).length > 0 &&
+                !_.some(_.map(contact[field.name], 'number'), (v) => _.isEmpty(v));
         default:
-            return !_.isEmpty(contact[field]);
+            return !_.isEmpty(contact[field.name]);
         }
     });
 }
 
+export function getMinRequiredFieldLabels() {
+    return LOOKUP_FIELDS.map((f) => gettext(f.label)).join(', ');
+}
+
+export function getMinRequiredFieldLabel(field) {
+    let thisField = LOOKUP_FIELDS.find((f) => (_.get(f, 'name') === field));
+
+    return _.get(thisField, 'label');
```

Comment:
You can use (Nullish coalescing operator)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator] instead of lodash functions.

Why do you use "label" as a default instead of logging an error if it can't be found?

Reply:
Sadly I'm not working in the develop branch as I would need to build a new environment. I don't think Nullish coalescing operator is available to me back in 1.31.

"label" is not a default it is the path in the object, I've added an empty string default. I think logging an error is reasonable.